assets/sass/components/analytics-4/_googlesitekit-analytics-setup-module.scss

```
@@ -17,19 +17,44 @@
  */
 
 .googlesitekit-plugin {
-
 	.googlesitekit-setup-module--analytics {
+		.googlesitekit-setup-module__inputs {
+			align-items: start;
+			.googlesitekit-setup__step-hint {
+				margin: 7px 0 0 15px;
+				padding: 0;
+			}
+		}
 
-		.googlesitekit-analytics-enable + .googlesitekit-setup-module__inputs {
+		.googlesitekit-setup-module__select_account {
+			font-size: $fs-body-lg;
+			margin-bottom: 0;
+		}
+
+		.googlesitekit-analytics-enable-enhanced-measurement__already-enabled-label {
+			font-size: $fs-body-sm;
+		}
 
+		.googlesitekit-module-settings-group__helper-text {
+			color: $c-surfaces-on-surface-variant;
+			font-size: $fs-body-sm;
+			margin: 4px 0;
+		}
+
+		.googlesitekit-color--surfaces-on-background-variant {
+			color: $c-surfaces-on-surface;
+			font-size: $fs-body-sm;
+			margin: 23px 0 0;
```

I believe this should also have a letter spacing of 0.2px.


```suggestion
			color: $c-surfaces-on-surface;
			font-size: $fs-body-sm;
			letter-spacing: $ls-xs;
			margin: 23px 0 0;
```